app/controllers/groups_controller.rb

```diff
@@ -411,6 +411,18 @@ def invite_member
       to_invite = params[:invite_member].split(',')
       errors = @grouping.invite(to_invite)
       if errors.blank?
+        if !to_invite.instance_of?(Array)
```

Comment:
I'm actually not sure if `Grouping#invite` is called from other places (you could check this) but in this case `.split(',')` always returns a list of strings, so you don't need this check here.